Remove AsyncStreaming client from python qps tests

kpayson64 opened the pull request:
Addresses #6810.  We have removed the python async bidirectional streaming client, so I've replaced those tests with a synchronous streaming client.

jtattermusch commented on tools/run_tests/performance/scenario_config.py:
```diff
@@ -413,8 +413,8 @@ def scenarios(self):
-        client_type='ASYNC_CLIENT', server_type='SYNC_SERVER',
+        'python_protobuf_sync_streaming_qps_unconstrained', rpc_type='STREAMING',
+        client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
         unconstrained_client='async')
```
unconstrained_client= needs to be 'sync' otherwise 6400 threads will be created?  Have you tried running all the python scenarios locally?

kpayson64 replied:
Yes, I've tried it locally.  The streaming client doesn't spin up a thread per outstanding request, it just queues up the number of outstanding requests before it starts reading.

jtattermusch commented:
Btw, I'm not sure the two scenarios getting fixed here are the only ones failing for python.

kpayson64 commented:
Updated the sync streaming client to properly follow the spec.

The issue I mentioned offline is actually not relevant here, I had an issue where more than 1 outstanding request was on a stream, which does not follow the ping pong semantics.  Once I fixed that, the issue went away.  Still needs investigation though.

jtattermusch commented:
LGTM on tests.
